IPython/utils/py3compat.py

```diff
@@ -172,9 +172,6 @@ def isidentifier(s, dotted=False):
             return all(isidentifier(a) for a in s.split("."))
         return s.isidentifier()
 
-    xrange = range
-    def iteritems(d): return iter(d.items())
-    def itervalues(d): return iter(d.values())
```

Comment:
Thanks a ton. Then i will keep this file as is. Will gradually change all projects references of module py3compat.py to py3 function calls,  then i think about remove it.